modules/system/assets/ui/js/select.js

```diff
@@ -113,6 +113,7 @@
             var placeholder = $element.data('placeholder')
             if (placeholder) {
                 extraOptions.placeholder = placeholder
+                extraOptions.allowClear = true;
```

Comment:
Please remove semicolon ;)

Reply:
Oops, done. Didn't pay attention that semicolons are not used there